storage/plugins/mobile-api/src/Controllers/CatalogController.php

```
@@ -61,12 +62,21 @@ public function search(
+            $requestedSort = isset($params['sort']) ? trim((string) $params['sort']) : null;
+            // Relevance is the default for a text query and can be selected
+            // explicitly by mobile clients. Facet-only browsing keeps the
+            // established newest/oldest/title/author keyset sorts.
+            $useRelevance = $query !== ''
+                && ($requestedSort === null || $requestedSort === '' || $requestedSort === 'relevance');
+            $sort = $useRelevance
+                ? ['key' => 'relevance', 'col' => 'relevance_score', 'dir' => 'DESC',
+                   'order' => 'relevance_score DESC, l.titolo ASC, l.id ASC', 'field' => 'relevance_score']
+                : $this->sortSpec($requestedSort);
+            $relevance = $useRelevance
+                ? SearchIndexBuilder::buildRelevanceOrder($this->db, $query, 'l.')
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SearchIndexBuilder' in method 'search'. (undefined)

(StaticAccess)